pkg/types/powervs/validation/platform_test.go

```diff
@@ -3,174 +3,170 @@ package validation
+			name:     "minimal",
+			platform: validMinimalPlatform(),
+			valid:    true,
 		},
 		{
 			name: "invalid region",
-			platform: &aws.Platform{
-				Region: "",
-			},
-			expected: `^test-path\.region: Required value: region must be specified$`,
+			platform: func() *powervs.Platform {
+				p := validMinimalPlatform()
+				p.Region = "invalid"
+				return p
+			}(),
+			valid: false,
 		},
 		{
-			name: "hosted zone with subnets",
-			platform: &aws.Platform{
-				Region:     "us-east-1",
-				Subnets:    []string{"test-subnet"},
-				HostedZone: "test-hosted-zone",
-			},
+			name: "missing region",
+			platform: func() *powervs.Platform {
+				p := validMinimalPlatform()
+				p.Region = ""
+				return p
+			}(),
+			valid: false,
 		},
 		{
-			name: "hosted zone without subnets",
-			platform: &aws.Platform{
-				Region:     "us-east-1",
-				HostedZone: "test-hosted-zone",
-			},
-			expected: `^test-path\.hostedZone: Invalid value: "test-hosted-zone": may not use an existing hosted zone when not using existing subnets$`,
-		},
-		{
-			name: "invalid url for service endpoint",
-			platform: &aws.Platform{
-				Region: "us-east-1",
-				ServiceEndpoints: []aws.ServiceEndpoint{{
-					Name: "ec2",
-					URL:  "/path/some",
-				}},
-			},
-			expected: `^test-path\.serviceEndpoints\[0\]\.url: Invalid value: "(.*)": host cannot be empty, empty host provided$`,
-		},
-		{
-			name: "invalid url for service endpoint",
-			platform: &aws.Platform{
-				Region: "us-east-1",
-				ServiceEndpoints: []aws.ServiceEndpoint{{
-					Name: "ec2",
-					URL:  "https://test-ec2.random.local/path/some",
-				}},
-			},
-			expected: `^test-path\.serviceEndpoints\[0\]\.url: Invalid value: "(.*)": no path or request parameters must be provided, "/path/some" was provided$`,
+			name: "valid machine pool",
+			platform: func() *powervs.Platform {
+				p := validMinimalPlatform()
+				p.DefaultMachinePlatform = validMachinePool()
+				return p
+			}(),
+			valid: true,
 		},
 		{
-			name: "invalid url for service endpoint",
-			platform: &aws.Platform{
-				Region: "us-east-1",
-				ServiceEndpoints: []aws.ServiceEndpoint{{
-					Name: "ec2",
-					URL:  "https://test-ec2.random.local?foo=some",
-				}},
-			},
-			expected: `^test-path\.serviceEndpoints\[0\]\.url: Invalid value: "(.*)": no path or request parameters must be provided, "/\?foo=some" was provided$`,
+			name: "valid vpc config",
+			platform: func() *powervs.Platform {
+				p := validMinimalPlatform()
+				p.VPC = "valid-vpc-name"
+				p.VPCZone = validVPCZone
+				p.Subnets = []string{"valid-compute-subnet-id", "valid-control-subnet-id"}
+				p.PowerVSResourceGroup = "rg-name"
+				return p
+			}(),
+			valid: true,
 		},
 		{
-			name: "valid url for service endpoint",
-			platform: &aws.Platform{
-				Region: "us-east-1",
-				ServiceEndpoints: []aws.ServiceEndpoint{{
-					Name: "ec2",
-					URL:  "test-ec2.random.local",
-				}},
-			},
+			name: "invalid vpc config missing vpc",
```

Comment:
for these names, can you add some punctuation? "invalid vpc: config missing vpc" or something like that?

Reply:
fixed